src/apps/workshop/workshop-app2/module/workshop2-autogen-openapi/src/openapi-schema/schemas/MetadataSummarySchema.yaml

```diff
@@ -10,9 +10,15 @@ properties:
     items:
       type: string
     description: list of all the supported schemas in this GeoNetwork system
-    example: ['iso19115-3.2018', 'iso19139']
+    example: [
+    "iso19115-3.2018",
+    "iso19139"
+    ]
   recordsPerSchema:
+    type: object
     additionalProperties:
       type: integer
     description: map from the schemaId (schema name) to the number of records of that schema type in this GeoNetwork system
-    example: "{'iso19115-3.2018': 4, 'iso19139': 39 }"
+    example:
+      iso19115-3.2018: 4
+      iso19139: 39
```

Comment:
I think it is fine to add examples, and if the examples are limited due to restrictions on `yaml` document ... that is fine we can talk about the restrictions.
(it is a workshop on what works now, highlighting limitations is *fine*)